sigstore/_cli.py

```diff
@@ -9,10 +9,21 @@ def main():
 
 
 @main.command("sign")
-def _sign():
-    click.echo(sign())
+@click.option("identity_token", "--identity-token", type=click.STRING)
+@click.argument("file_", metavar="FILE", type=click.File("r"), required=True)
+def _sign(file_, identity_token):
+    click.echo(sign(file_=file_, identity_token=identity_token, output=click.echo))
 
 
 @main.command("verify")
-def _verify():
-    click.echo(verify())
+@click.option("certificate_path", "--cert", type=click.Path())
```

Comment:
Just confirming. Is the idea here that we can supply a certificate and signature to verify "offline". But if we don't supply both of these, we check against Rekor?

Reply:
We discussed this offline!